student.rb

```diff
@@ -6,6 +6,7 @@ class Student < Person
   def initialize(age, classroom, name, parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    @classroom.add_student(self)
```

Comment:
- [ ] This is not what we want to establish the relationship between student and classroom. The association should be added through its own method. Kindly remove this line of code.

  ```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```